openqa-label-known-issues

```diff
@@ -136,31 +138,33 @@ investigate_issue() {
     echo "$reason" >> "$out"
     if [[ "$curl_output" != "200" ]] && [[ "$curl_output" != "301" ]]; then
         # if we can not even access the page it is something more critical
-        handle_unreachable "$testurl" "$out"
-        # Assume autoinst-log.txt still not found but process job anyway due to reason var
-        if [[ -n $reason ]] && [[ $curl_output = "404" ]] && [[ $reason != null ]]; then
-            label_on_issues_from_issue_tracker "$id"
+        handle_unreachable "$testurl" "$out" || return
+        # If we something other than a 404, we should not continue
```

Comment:
This comment is not grammatically correct. I would just remove it completely because it does not add any information anyway. (The comment just repeats a simple line of code anybody can just read directly.)

```suggestion
```

Reply:
@b10n1k Did you see this?

Reply:
Agreed. @b10n1k Please just drop the comment.